app/services/rummager.rb

```diff
@@ -8,8 +7,7 @@ def self.rummager
 
   class Rummager
     def search(args)
-      results = rummager.search(args)
-      SearchMutator.mutate_search(results)
+      rummager.search(args)
```

Comment:
(We could do away with this decorator now that we're not actually decorating it; although I'd thought about leaving it in anyway, because it might be nice to add to later)